frontend/packages/topology/src/utils/application-utils.ts

```diff
@@ -131,7 +130,11 @@ const safeKill = async (model: K8sKind, obj: K8sResourceKind) => {
     namespace: obj.metadata.namespace,
   });
   if (resp.status.allowed) {
-    return k8sKill(model, obj);
+    try {
+      return await k8sKill(model, obj);
+    } catch (ignored) {
+      // 404 expected
+    }
```

Comment:
If would expect if this would check for the real status code. Maybe something like that? (untested code!)

```suggestion
    } catch (error) {
      if (error.response?.status === 404) {
        // ignore resource which could not be found.
      } else {
        throw error;
      }
    }
```